cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookReport.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.cristin.mapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@JsonIgnoreProperties({"hefte", "utgave", "utgave_fra", "doi", "status_revidert", "status_elektronisk_publisert",
+        "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
+        "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
+        "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
+        "forlag", "fagfelt"})
+public class CristinBookReport {
+
+    public static final String ISBN_LIST = "isbn";
+    public static final String PUBLISHER = "utgivernavn";
+    public static final String NUMBER_OF_PAGES = "antall_sider_totalt";
+
+    @JsonProperty(ISBN_LIST)
+    private String isbn;
+    @JsonProperty(PUBLISHER)
+    private String publisherName;
+    @JsonProperty(NUMBER_OF_PAGES)
+    private String numberOfPages;
+
+    public CristinBookReport() {
+
+    }
+
```

Comment:
add a copy method that returns a builder. see CrstinObject.

Reply:
Done